wltourney/static/js/tournament.js

```diff
@@ -578,10 +578,6 @@ $(function () {
                           {
                                $("#gamelogtab").html(data.game_log);
                                $('#game_log_data_table').DataTable();
-
-                               $('.time_to_boot').each(function () {
```

Comment:
for my education what's the reason this isn't needed?

Reply:
I removed the time to boot column.

Reply:
Nice.